From c7e4747c1589b2d090361cb753f212a7eec5ca7f Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Sat, 19 Nov 2011 15:59:21 -0500
Subject: [PATCH 1/2] Makefile: fix install target with out-of-tree builds

The install target always looks for the compiled binaries in the working
directory which breaks when doing something like `make CHOST=... install`.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 Makefile |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index d0bc3d6..4b71b8f 100644
--- a/Makefile
+++ b/Makefile
@@ -58,9 +58,9 @@ endif
 	rm -f $(BUILDDIR)/include/version.h
 	$(MAKE) -C $(TESTS) clean
 
-install:: ${BINS} ${SCRIPTS}
+install:: $(addprefix $(BUILDDIR)/,${BINS}) ${SCRIPTS}
 	mkdir -p ${DESTDIR}/${SBINDIR}
-	install -m 0755 ${BINS} ${SCRIPTS} ${DESTDIR}/${SBINDIR}/
+	install -m 0755 $^ ${DESTDIR}/${SBINDIR}/
 	mkdir -p ${DESTDIR}/${MANDIR}/man1
 	gzip -9c mkfs.jffs2.1 > ${DESTDIR}/${MANDIR}/man1/mkfs.jffs2.1.gz
 
-- 
1.7.6.1

